tier-s3.go

```diff
@@ -23,14 +23,17 @@ package madmin
 
 // TierS3 represents the remote tier configuration for AWS S3 compatible backend.
 type TierS3 struct {
-	Endpoint     string `json:",omitempty"`
-	AccessKey    string `json:",omitempty"`
-	SecretKey    string `json:",omitempty"`
-	Bucket       string `json:",omitempty"`
-	Prefix       string `json:",omitempty"`
-	Region       string `json:",omitempty"`
-	StorageClass string `json:",omitempty"`
-	AWSRole      bool   `json:",omitempty"`
+	Endpoint           string `json:",omitempty"`
+	AccessKey          string `json:",omitempty"`
+	SecretKey          string `json:",omitempty"`
+	Bucket             string `json:",omitempty"`
+	Prefix             string `json:",omitempty"`
+	Region             string `json:",omitempty"`
+	StorageClass       string `json:",omitempty"`
+	AWSRole            bool   `json:",omitempty"`
+	AWSRoleARN         string `json:",omitempty"`
+	AWSRoleSessionName string `json:",omitempty"`
+	DurationSeconds    int    `json:",omitempty"`
```

Comment:
this is not used?

Reply:
I think I need to add an option pattern for it.